### PR TITLE
Added Word Embeddings (Word2Vec)

### LINKED ISSUE #3: Add Word Embeddings
**Name**
Word Embeddings (Word2Vec)

**Packages used**
tensorflow==2.5.0
numpy ==1.19.5
pandas==1.2.4

**Brief  explanation**
Word embedding is used to extract semantic relations between different words by plotting them in a vector space.

### PR DESCRIPTION
**Pull request by r-o-n-i-t**
fixes #3 

# Description
Word Embeddings (Word2Vec)

## Type of change

Note: Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation update
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] New layer/algorithm
- [ ] Other (Specify)

### Implementation details
(only for `New layer/algorithm`)
> * List all the packages and their versions used.
tensorflow==2.9.2
re==2.2.1

> * Include test results:  
The model reaches a validation accuracy of around 80%

## Checklist

- [X] I have performed a self code_review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by mayureshagashe2105 — CHANGES_REQUESTED** [review]
Hey @r-o-n-i-t, thank you for making the suggested changes, looks good to me. 
One small request from our side
> can you please provide a README in the same folder so that others can get a clearer picture? Give a brief explanation about the algorithm and its working. You can look at other algo directories to get an idea about the README. 

Thank you for your patience will merge after you add README. Cheers!

**Review by HemanthSai7 — COMMENTED** [review]
Hi @r-o-n-i-t , Thank you for the PR. But as I said before we have to implement the algorithms from scratch. You have used TensorFlow to implement this algorithm.

**Comment by r-o-n-i-t** [comment]
> Hi @r-o-n-i-t , Thank you for the PR. But as I said before we have to implement the algorithms from scratch. You have used TensorFlow to implement this algorithm. Please look into other algorithms already implemented in the repo to understand the requirement of the project. Thanks!

Thank you for the reference I guess I have understood the task this time 😅.
Please find the new PR and review it accordingly.
Thank you
